crates/configuration/src/version5/metadata/native_operations.rs

```diff
@@ -286,8 +286,11 @@ pub fn parse_native_query_from_file(
 }
 
 /// Parse a native query into parts where variables have the syntax `{{<variable>}}`.
-pub fn parse_native_query(string: &str) -> NativeQueryParts {
-    let vec: Vec<Vec<NativeQueryPart>> = string
+pub fn parse_native_query(string_untrimmed: &str) -> NativeQueryParts {
+    let string_trimmed = string_untrimmed.trim_end();
+    let vec: Vec<Vec<NativeQueryPart>> = string_trimmed
+        .strip_suffix(';')
+        .unwrap_or(string_trimmed)
         .split("{{")
         .map(|part| match part.split_once("}}") {
```

Comment:
Any reason not to use `.flat_map` here?

Reply:
Not really. I don't mind doing that as part of this PR if you wish.